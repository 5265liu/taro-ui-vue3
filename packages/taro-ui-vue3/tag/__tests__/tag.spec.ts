import { mount } from '@vue/test-utils'
import AtTag from '../src/index.vue'

const AXIOM = 'Rem is the best girl'

describe('AtTag.vue', () => {
  test('render test', () => {
    const wrapper = mount(AtTag, {
      slots: {
        default: AXIOM,
      },
    })
    expect(wrapper.text()).toEqual(AXIOM)
  })
})
