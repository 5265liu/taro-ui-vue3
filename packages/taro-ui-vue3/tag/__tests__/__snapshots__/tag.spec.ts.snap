// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AtTag Snap render AtTag -- props active 1`] = `
<view
  class="at-tag--normal at-tag--active at-tag"
>
  
  标签
  
</view>
`;

exports[`AtTag Snap render AtTag -- props circle 1`] = `
<view
  class="at-tag--normal at-tag--circle at-tag"
>
  
  标签
  
</view>
`;

exports[`AtTag Snap render AtTag -- props disabled 1`] = `
<view
  class="at-tag--normal at-tag--disabled at-tag"
>
  
  标签
  
</view>
`;

exports[`AtTag Snap render AtTag -- props name 1`] = `
<view
  class="at-tag--normal at-tag"
>
  
  标签
  
</view>
`;

exports[`AtTag Snap render AtTag -- props size 1`] = `
<view
  class="at-tag--small at-tag"
>
  
  标签
  
</view>
`;

exports[`AtTag Snap render AtTag -- props type 1`] = `
<view
  class="at-tag--normal at-tag--primary at-tag"
>
  
  标签
  
</view>
`;
